src/chapters/unit-testing/tdd.rst

```diff
@@ -0,0 +1,102 @@
+Test-Driven Development
+=======================
+We all want to write code that *works*! How do you know if your code works? You can
```

Comment:
This paragraph is somewhat redundant to the first section of the chapter. Rephrase so as to emphasize the points and remind students of them, rather than stating them for the first time.